wsclient/src/main/java/org/hpccsystems/ws/client/platform/DFUDataColumnInfo.java

```diff
@@ -76,7 +91,7 @@ public String toString()
      * 
      * @param base
      */
-    protected void copy(DFUDataColumn base)
+    protected void copy(org.hpccsystems.ws.client.gen.wsdfu.v1_36.DFUDataColumn base)
```

Comment:
are we losing the ability to copy pre/post 1.36 DFUDatacolumns ? do we need other copy methods which accept other versions of DFUDataColumn ?